rmbproxy/models.go

```diff
@@ -0,0 +1,47 @@
+package rmbproxy
+
+import (
+	"github.com/patrickmn/go-cache"
+	"github.com/threefoldtech/zos/pkg/substrate"
+)
+
+// DefaultExplorerURL is the default explorer graphql url
+const DefaultExplorerURL string = "https://explorer.devnet.grid.tf/graphql/"
+
+// App is the main app objects
+type App struct {
+	resolver TwinResolver
+	lruCache *cache.Cache
+}
+
+type Message struct {
```

Comment:
data type definitions should be reused not copied https://github.com/threefoldtech/rmb-go/blob/master/server.go (speaking of which both repos should have pkg directory for things that are reusable from elsewhere)

Reply:
Actually, it's not necessary to have this dependency at all. The rmb message type is subject to change and the proxy doesn't have to assume anything about the type the rmb expects. It can just receive bytes, forward them to the rmb, and return the string response.